R/create.R

```diff
@@ -59,6 +59,8 @@ create_project <- function(path = ".",
 
   if (rstudio) {
     use_rstudio()
+  } else {
+    writeLines(character(), file.path(path, ".here"))
```

Comment:
Probably should eventually extract out into `use_here()` or similar, but no need for this PR.

Reply:
Yeah. That brings up whether usethis should embrace the here package in addition to rprojroot. As in, use rprojroot internally but deal more with here for user-facing stuff.